city_scrapers/spiders/chi_board_elections.py

```diff
@@ -0,0 +1,167 @@
+                'note': ''
+            }],
+        }
+
+        data['status'] = self._generate_status(data)
+        data['id'] = self._generate_id(data)
+        yield data
+
+    def _prev_meetings(self, response):
+        meetings = response.xpath("//a/text()").extract()
+        meetingdates = [
+            meeting[22:]
+            for meeting in meetings
+            if "Minutes" not in meeting and "mode" not in meeting and meeting is not " "
+        ]
+        for meetingdate in meetingdates:
+            meetingdate = "9:30 a.m. on " + meetingdate
+            data = {
+                '_type': 'event',
+                'name': "Chicago Board of Election Commissioners",
+                'event_description': "Meeting",
+                'classification': COMMISSION,
+                'start': self._parse_start(meetingdate),
+                'end': {},
+                'all_day': False,
+                'location': self._parse_location(response),
+                'documents': [],
+                'sources': [{
+                    'url':
+                        "https://app.chicagoelections.com/pages/en/meeting-minutes-and-videos.aspx",
+                    'note': ''
+                }],
+            }
+
+            data['status'] = self._generate_status(data)
+            data['id'] = self._generate_id(data)
+
+            yield data
+
+    def _parse_next(self, response):
+        """
+        Get next page. You must add logic to `next_url` and
+        return a scrapy request.
+        """
+        next_url = None  # What is next URL?
+        return scrapy.Request(next_url, callback=self.parse)
+
+    def _parse_name(self, item):
+        """
+        Parse or generate event name.
+        """
+        return ''
+
+    def _parse_description(self, item):
+        """
+        Parse or generate event description.
+        """
+        return ''
+
+    def _parse_classification(self, item):
+        """
+        Parse or generate classification (e.g. public health, education, etc).
+        """
+        return ''
+
+    def _parse_start(self, item):
+        """
+        Parse start date and time.
+        """
+        formatitem = item.replace("a.m.", "AM")
+        formatitem = formatitem.replace("p.m.", "PM")
+        formatitem = formatitem.replace("Sept", "Sep")
+        try:
+            datetime_item = datetime.strptime(formatitem, '%I:%M %p on %b. %d, %Y')
+        except ValueError:  # Some months are abbreviated, some are not
+            datetime_item = datetime.strptime(formatitem, '%I:%M %p on %B %d, %Y')
+        dict = {'date': datetime_item.date(), 'time': datetime_item.time(), 'note': ''}
+        return dict
+
+    def _parse_end(self, item):
+        """
+        Parse end date and time.
+        """
+        return ''
+
+    def _parse_all_day(self, item):
+        """
+        Parse or generate all-day status. Defaults to False.
+        """
+        return False
+
+    def _parse_location(self, item):
```

Comment:
This works, but we should add a sanity check here that looks for `69 W Washington` (more flexible than that though) and returns an error if it isn't in the description. That way we'll know when/if the location changes. This likely only applies to `_next_meeting` though